spec/features/cli_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'aruba/rspec'
 
 RSpec.describe GemfileLocker::CLI, type: :aruba do
-  subject { -> { run_simple "gemfile_locker #{action} #{args}" } }
+  subject { -> { run_command_and_stop "gemfile_locker #{action} #{args}" } }
```

Comment:
Is this still required?

Reply:
Nice question. I only fixed the deprecated error